dot/parachain/availability-store/availability_store.go

```diff
@@ -68,11 +167,34 @@ func (as *AvailabilityStore) loadAvailableData(candidate common.Hash) (*Availabl
 	return &result, nil
 }
 
+// writeAvailableData put available data in the availabilityBatch of the given batch
+func (as *AvailabilityStore) writeAvailableData(batch *AvailabilityStoreBatch,
+	candidate parachaintypes.CandidateHash, data AvailableData) error {
+	dataBytes, err := json.Marshal(data)
```

Comment:
is it possible to use `scale.Marshal` here?

Reply:
Yes